packages/extension/src/ui/features/actions/transaction/useTransactionSimulatedData.ts

```diff
@@ -65,6 +65,41 @@ export type ValidatedTokenApproval = Omit<
   usdValue: string | undefined
 }
 
+function partitionIncomingOutgoingTransfers(transfers: AggregatedSimData[]) {
+  return partition(transfers, (t) => t.amount.isGreaterThan(0))
+}
+
+function orderAggregatedSimData(
+  simData: AggregatedSimData[],
+): AggregatedSimData[] {
+  const orderedSimData = orderBy(simData, (t) => t.amount.toString(16), "desc")
+
+  const [erc721Transfers, restTransfers] = partition(
+    orderedSimData,
+    (t) => t.token.type === "erc721",
+  )
+  const [erc1155Transfers, erc20Transfers] = partition(
+    restTransfers,
+    (t) => t.token.type === "erc1155",
+  )
```

Comment:
```suggestion
  const [erc20Transfers, erc1155Transfers] = partition(
    restTransfers,
    (t) => t.token.type === "erc20",
  )
```

No support for erc1155 for now. Better to do it this way

Reply:
Sure, I feel erc20 partitioning makes more sense but sure your works too.

Reply:
I think you're right, lets have 
1. erc721 transfers
2. all other transfers (including 1155)
3. erc20 transfers